solutions/instances/variables.tf

```diff
@@ -251,10 +251,10 @@ variable "kms_region" {
   description = "The region in which KMS instance exists."
 }
 
-variable "existing_kms_guid" {
+variable "existing_kms_crn" {
   type        = string
   default     = null
-  description = "The GUID of of the KMS instance used for the COS bucket root Key. Only required if not supplying an existing KMS root key and if 'skip_cos_kms_auth_policy' is true."
+  description = "The CRN of of the KMS instance used for the COS bucket root Key. Only required if not supplying an existing KMS root key and if 'skip_cos_kms_auth_policy' is true."
```

Comment:
`of of ` -> of
`and if 'skip_cos_kms_auth_policy' is true` - can be deleted, as we are passing it true always